### PR TITLE
Add fast post processing

### PR DESCRIPTION
**Pull request by Daniil-Osokin**
Added fast post-processing from OpenVINO implementation. @druzhkov-paul Many thanks for making post-processing faster in times!

### CODE REVIEW

File: modules/keypoints.py [hunk 1]
```diff
@@ -48,111 +48,74 @@ def extract_keypoints(heatmap, all_keypoints, total_keypoint_num):
...
-                cur_point_score = (vec[0] * part_pafs[mid_point[0][1], mid_point[0][0], 0] +
-                                   vec[1] * part_pafs[mid_point[1][1], mid_point[1][0], 1])
-
-                height_n = pafs.shape[0] // 2
-                success_ratio = 0
-                point_num = 10  # number of points to integration over paf
-                if cur_point_score > -100:
-                    passed_point_score = 0
-                    passed_point_num = 0
-                    x, y = linspace2d(kpt_a, kpt_b)
```
Comment (druzhkov-paul) [inline modules/keypoints.py:119]: `linspace2d` seems to be redundant now.

File: modules/keypoints.py [hunk 2]
```diff
@@ -48,111 +48,74 @@ def extract_keypoints(heatmap, all_keypoints, total_keypoint_num):
...
-                    for point_idx in range(point_num):
-                        if not demo:
```
Comment (druzhkov-paul) [inline modules/keypoints.py:121]: Updated code does not depend on the demo mode.

**Comment by druzhkov-paul** [comment]
Great to see that you've found the code useful.

File: modules/keypoints.py (hunk 1 above)
Comment (Daniil-Osokin) [reply modules/keypoints.py:119]: Right! Thanks for the review!

**Comment by Daniil-Osokin** [comment]
:accept:

**Pull request merged**